config/environments/development.rb

```diff
@@ -6,17 +6,20 @@
   # In the development environment your application's code is reloaded any time
   # it changes. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
-  config.cache_classes = false
+  config.enable_reloading = true
```

Comment:
Actually now that I checked here https://guides.rubyonrails.org/v7.0/configuring.html I think this config doesn't exist in Rails 7.0.4 🤔